multi-casskop/go.mod

```diff
@@ -7,30 +7,44 @@ require (
 	admiralty.io/multicluster-service-account v0.6.0
 	cloud.google.com/go v0.46.3 // indirect
 	github.com/Orange-OpenSource/casskop v0.4.1
-	github.com/gogo/protobuf v1.3.0 // indirect
-	github.com/googleapis/gnostic v0.3.1 // indirect
 	github.com/gregjones/httpcache v0.0.0-20190611155906-901d90724c79 // indirect
-	github.com/hashicorp/golang-lru v0.5.3 // indirect
 	github.com/imdario/mergo v0.3.8
 	github.com/jessevdk/go-flags v1.4.0
 	github.com/kylelemons/godebug v1.1.0
-	github.com/operator-framework/operator-sdk v0.10.0
+	github.com/operator-framework/operator-sdk v0.15.0
```

Comment:
Did you see 0.15.1 is out ?

Reply:
Yes, this version is just a fix on ansible operator part. So no need to go to upgrade for the moment.